htdocs/components/99_announcement_banner.css

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+ * Copyright [2016-2018] EMBL-European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#announcement-banner {
+    background-color: #CD0000;
+    padding: 12px 8px 12px 8px;
+    color: #FFF;
+    text-align: center;
+}
```

Comment:
@ens-st3 I think Steve will decide the default colours for the banner on live site.

Reply:
@tuanebi I sent the screenshot to Steve with the new colours we tried and he has Okayed it.

Reply:
@msmanoj - can you confirm that we're not showing the pop up window ? That is to only be used in the few days before tools falls back to HH, everything else is to be communicated by the masthead banner

Reply:
@ens-st3 Yes, I confirm that we are not displaying the pop up window. Also, the banner will start to appear only after creating the temporary file with the banner content, which we will be doing tomorrow.